dictionaries/en_US/src/legacy/en_US.txt

```diff
@@ -23247,3 +23247,5 @@ zymoplastic
 zymoscope
 zyzzyva
 zyzzyvas
+DRM # Digital rights management
```

Comment:
Why add these here? I prefer no new additions to legacy. The meaning of legacy is that it is a legacy list, not a list of legacy terms.

Reply:
OK, noted.

I gave the reason in the commit.
I thought I could do it,so I did 😅

Maybe you will be OK if I remove DRM and IFTTT from en_US as they are available in software-terms dictionary.

Otherwise I can either add move them in another file in en_US, or add a comment about them in en_US/src/acronyms.txt

Reply:
I chose to let them here for now

Reply:
There is not need to normalize the dictionaries. Having the same word in multiple places is ok.

Please note: Not everyone is using the software-terms dictionary. It is currently enabled by default, but I hope to turn that off and make it dependent upon the environment.